python/torch_mlir/dialects/torch/importer/jit_ir/csrc/CMakeLists.txt

```diff
@@ -30,6 +30,7 @@ target_link_libraries(TorchMLIRJITIRImporter
 
 message(STATUS "TORCH_CXXFLAGS=${TORCH_CXXFLAGS}")
 set_target_properties(TorchMLIRJITIRImporter PROPERTIES
+  INSTALL_PREFIX python_packages/torch_mlir/torch_mlir/_mlir_libs
```

Comment:
why is this needed here?

Reply:
It wasn't meant to be in this PR :-( I've removed it.